docs/api/cozy-pouch-link.md

```diff
@@ -105,6 +106,24 @@ and return if those queries are already warmedup or not
 | --- | --- |
 | doctype | <code>string</code> | 
 
+<a name="PouchLink+mergePartialIndexInSelector"></a>
+
+### pouchLink.mergePartialIndexInSelector(selector, partialFilter) ⇒ <code>object</code>
+Merge the selector and partial filter definitions.
+
+This is necessary because PouchDB does not support partial indexes, so
+we have to express any partial filter definition directly in the selector.
+Note the partial filter evaluation will be done in-memory, which might decrease
+performances.
+
+**Kind**: instance method of [<code>PouchLink</code>](#PouchLink)  
+**Returns**: <code>object</code> - The merge between the selector and the partial filter  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| selector | <code>object</code> | The query selector |
+| partialFilter | <code>object</code> | The partial filter |
+
```

Comment:
I am wondering if this should be exposed in the doc as it seems like a private method ?